pkg/question/question.go

```diff
@@ -1188,3 +1196,19 @@ func AskTerminationConfirmation(instanceIds []string) string {
 
 	return answer
 }
+
+func AskCapacityType() string {
+	question := fmt.Sprintf("Select capacity type. Spot instances are available at up to a 90%% discount compared to On-Demand instances,\nbut they may get interrupted by EC2 with a 2-minute warning")
```

Comment:
Extra percent sign on `90%%`

Reply:
Thats not an extra percent. Thats just how its formatted

Reply:
Ohhh right, because it's a "printf" :) That makes sense.

In this case since you're not using any substitution into the string, you could supply a simple string literal, but this is also fine.